addOns/pscanrules/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - The Big Redirect scan rule will now also alert on responses that have multiple HREFs (idea from xnl-h4ck3r).
     - It also now includes example alert and alert reference functionality for documentation generation purposes (Issues 6119, 7100, and 8189).
 - Update reference for X-Content-Type-Options Header Missing and Content-Type Header Missing (Issue 8262).
+- Update Vulnerabilities' references to use https links for Loosely Scoped Cookie (Issue 8262).
```

Comment:
Add to the above line instead.

Reply:
I hope I've understood correctly - ok then?